src/app/clusters/webrtc-transport-requestor-server/BUILD.gn

```diff
@@ -11,5 +11,22 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-group("webrtc-transport-requestor-server") {
+import("//build_overrides/build.gni")
+import("//build_overrides/chip.gni")
+
+source_set("webrtc-transport-requestor-server") {
+  sources = [
+    "webrtc-transport-requestor-cluster.cpp",
+    "webrtc-transport-requestor-cluster.h",
+  ]
+
+  public_deps = [
+    "${chip_root}/src/app:attribute-access",
+    "${chip_root}/src/app/data-model-provider:metadata",
+    "${chip_root}/src/app/server-cluster",
+    "${chip_root}/src/lib/core:error",
+    "${chip_root}/src/lib/support",
+    "${chip_root}/zzz_generated/app-common/clusters/WebRTCTransportRequestor",
+  ]
+  public_configs = [ "${chip_root}/src:includes" ]
```

Comment:
is this needed? To not have it everywhere I wonder if just relying on transitive configs is sufficient.

Reply:
"${chip_root}/zzz_generated/app-common/clusters/WebRTCTransportRequestor", is still needed to pass compile, others were removed